src/transformation/visitors/jsx/xhtml.ts

```diff
@@ -0,0 +1,257 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const entities: { [name: string]: string } = {
```

Comment:
Why not just do this? Instead of declaring the export separately? (I think you could also drop the eslint disable?)
```suggestion
export const XHTMLEntities: { [name: string]: string } = {
```